src/OpenTelemetry/CHANGELOG.md

```diff
@@ -2,12 +2,17 @@
 
 ## Unreleased
 
+* Throw NotSupportedException when using `ExceptionProcessor` for Tracing in
+  Mono Runtime and Native AOT environment because the the dependent
```

Comment:
The generic `NotSupportedException` message would not mention anything about AOT.

Could you add `<remarks>` in the [code documentation ](https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Trace/Builder/TracerProviderBuilderExtensions.cs#L24) for `SetErrorStatusOnException` method saying "This method is not supported for AOT as of .NET 8."?

Reply:
done!